__tests__/app.test.js

```diff
@@ -3,10 +3,29 @@ const seed = require("../db/seeds/seed.js");
 const testData = require("../db/data/test-data");
 const request = require("supertest");
 const app = require("../app.js");
+const fs = require("fs/promises");
 
 beforeEach(() => seed(testData));
 afterAll(() => db.end());
 
+describe("/api", () => {
+  test("GET:200 responds with an object describing all endpoints available", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        const readPromise = fs.readFile(`${__dirname}/../endpoints.json`);
+
+        return Promise.all([readPromise, body]);
+      })
+      .then(([fileContents, body]) => {
+        const endpointsData = JSON.parse(fileContents);
+
+        expect(body).toMatchObject(endpointsData);
```

Comment:
Nicely done! We could simplify this though by just requiring in the .json file, then expecting the response to equal it 👍

Reply:
Oooh didn't realise I could just require it in instead of reading it! I'll refactor it to this to simplify it, thanks 👍